lib/link.js

```diff
@@ -41,6 +42,10 @@ export default class Link extends Component {
   }
 
   render () {
+    if (Children.count(this.props.children) > 1) {
+      warn(`Warning: 'Link' accepts only one child element.`)
+    }
```

Comment:
I agree, will change it 👍 So far I haven't found a PropType for a single React element, but the docs say `React.Children.only` will throw an error if more than one item is set. So I guess that's ok?